presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestInsertIntoHiveTable.java

```diff
@@ -39,23 +40,41 @@
 
 public class TestInsertIntoHiveTable
         extends ProductTest
+        implements RequirementsProvider
 {
     private static final String TABLE_NAME = "target_table";
+    private static final String PARTITIONED_TABLE_WITH_SERDE = "target_partitioned_with_serde_property";
 
-    private static class AllSimpleTypesTables
-            implements RequirementsProvider
+    @Override
+    public Requirement getRequirements(Configuration configuration)
     {
-        @Override
-        public Requirement getRequirements(Configuration configuration)
-        {
-            return compose(
-                    mutableTable(ALL_HIVE_SIMPLE_TYPES_TEXTFILE, TABLE_NAME, CREATED),
-                    immutableTable(ALL_HIVE_SIMPLE_TYPES_TEXTFILE));
-        }
+        return compose(
+                mutableTable(ALL_HIVE_SIMPLE_TYPES_TEXTFILE, TABLE_NAME, CREATED),
+                mutableTable(partitionedTableDefinition(), PARTITIONED_TABLE_WITH_SERDE, CREATED),
+                immutableTable(ALL_HIVE_SIMPLE_TYPES_TEXTFILE));
+    }
+
+    private static TableDefinition partitionedTableDefinition()
+    {
+        String createTableDdl = "CREATE TABLE %NAME%(" +
+                "id int," +
+                "name string" +
+                ")" +
+                "PARTITIONED BY (dt string)" +
+                "ROW FORMAT SERDE 'org.apache.hadoop.hive.serde2.lazy.LazySimpleSerDe'" +
```

Comment:
i didn't notice this previously, sorry. Add space (` `) at the end of every line. Otherwise this is `...SerDe'WITH SERDEPROPERTIES ( ...`

Reply:
Oh, sorry. Added spaces.